glustercli/cli/parsers.py

```diff
@@ -226,11 +229,19 @@ def parse_volume_info(info, group_subvols=False):
 
 
 def _parse_a_node(node_el):
-    brick_path = (node_el.find('hostname').text + ":" + node_el.find('path').text)
+    name = (node_el.find('hostname').text + ":" + node_el.find('path').text)
+    online = node_el.find('status').text == "1" or False
+    if not online:
+        # if the node where the brick exists isn't
+        # online then no reason to continue as the
+        # caller of this method will populate "default"
+        # information
+        return {'name': name, 'online': online}
```

Comment:
With this change, Also change here https://github.com/gluster/glustercli-python/blob/master/glustercli/cli/parsers.py#L290

```python
if brick_status_data is None or not brick_status_data.online:
```